src/providers/ldap/ldap_child.c

```diff
@@ -513,6 +526,7 @@ static krb5_error_code ldap_child_get_tgt_sync(TALLOC_CTX *memctx,
     *expire_time_out = my_creds.times.endtime - kdc_time_offset;
 
 done:
+    krb5_get_init_creds_opt_free(context, options);
```

Comment:
Well, I know it is actually [safe](https://github.com/krb5/krb5/blob/master/src/lib/krb5/krb/gic_opt.c#L164), but strictly speaking [docs](https://web.mit.edu/kerberos/krb5-devel/doc/appdev/refs/api/krb5_get_init_creds_opt_free.html#c.krb5_get_init_creds_opt_free) don't say explicitly if it is ok to pass NULL.
And also we do check other krb5 resources for NULL before disposing, so probably it makes sense to check `options` as well. At least for consistency...